tutormfe/templates/mfe/build/mfe/env.config.jsx

```diff
@@ -43,7 +43,7 @@ async function setConfig () {
     {%- endfor %}
 
     {{- patch("mfe-env-config-runtime-final") }}
-  } catch { }
+  } catch (error) { console.error("env.config.jsx failed to apply.";}
```

Comment:
Since we're using a catch-all, it would be useful to log the error message as well, I think.

Reply:
Sure. I misunderstood console.error that it would automatically log the error, it is not the case.